roles/openshift_docker_facts/tasks/main.yml

```diff
@@ -15,6 +15,9 @@
       disable_push_dockerhub: "{{ openshift_disable_push_dockerhub | default(None) }}"
       hosted_registry_insecure: "{{ openshift_docker_hosted_registry_insecure | default(None) }}"
       hosted_registry_network: "{{ openshift_docker_hosted_registry_network | default(None) }}"
+  - role: common
+    local_facts:
+      openshift_deployment_type: "{{ deployment_type }}"
```

Comment:
I think it can be removed. I was having a really hard time in the backbort
branch ensuring that this was set early on but I don't think master suffers
from those issues. Testing without it now.

On Fri, Apr 1, 2016 at 2:51 PM, Jason DeTiberus <notifications@github.com>
wrote:

> In roles/openshift_docker_facts/tasks/main.yml
> <https://github.com/openshift/openshift-ansible/pull/1655#discussion_r58250681>
> :
>
> > @@ -15,6 +15,9 @@
> >        disable_push_dockerhub: "{{ openshift_disable_push_dockerhub | default(None) }}"
> >        hosted_registry_insecure: "{{ openshift_docker_hosted_registry_insecure | default(None) }}"
> >        hosted_registry_network: "{{ openshift_docker_hosted_registry_network | default(None) }}"
> > +  - role: common
> > +    local_facts:
> > +      openshift_deployment_type: "{{ deployment_type }}"
>
> Why would we persist this as openshift.common.openshift_deployment_type?
> shouldn't it just be openshift.common.deployment_type?
>
> Also, this should be pushed down into openshift_facts, which is a
> dependency of openshift_docker_facts.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/openshift-ansible/pull/1655/files/dde08ef1549260499bdcb26f6a5125c2e485309f#r58250681>
>